tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -61,7 +55,7 @@ struct MklBatchNormFwdParams {
                         bool training)
       : src_dims(src_dims), depth(depth), eps(eps), training(training) {}
 
-#endif
+#endif // !ENABLE_MKLDNN_V1
```

Comment:
Nit: There are two spaces between `#endif` and `//`.
```suggestion
#endif  // !ENABLE_MKLDNN_V1
```

Reply:
Sorry to be nitpicking again, but the spacing still isn't right (only one space between `#endif` and `//`. There should be two spaces between `#endif` and `//`, i.e., `#endif<two spaces>//`. For example,
```suggestion
#endif  // !ENABLE_MKLDNN_V1
```

Reply:
Oops, I interpreted it as one space. So I changed all to have 1 space. Will change to add an additional space

Reply:
Thank you so much! :)